modules/nw-ovn-k-adminnetwork-policy.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn-k-network-policy.adoc
+
+:_mod-docs-content-type: <Concept>
+[id="AdminNetworkPolicy"_{context}"]
+= AdminNetworkPolicy
+
+`AdminNetworkPolicy` (ANP) is a cluster-scoped custom resource definitions (CRDs). As an {product-title} administrator, you can use `AdminNetworkPolicy` to secure your cluster in the following ways:
+
+* You can create cluster-scoped policies prior to namespaces being created.
+
+* You can create deny and allow traffic rules at a cluster level that are non-overridable by users in the cluster.
+
+The key difference between `AdminNetworkPolicy` and `NetworkPolicy` is that ANP provides cluster scoped policy for administrators while `NetworkPolicy` provides namespaced scoped policy for developers.
+
+An ANP allows administrators to specify:
+
+* A `subject` that consists of a set of pods that is selected by the policy
+
+* A list of ingress rules to be applied for all ingress traffic towards the `subject`.
+
+* A list of egress rules to be applied for all egress traffic from the `subject`.
```

Comment:
what about priorities in ANP? Are you covering that elsewhere? it also allows to specific a priority...